dev/ci/user-overlays/README.md

```diff
@@ -1,38 +1,72 @@
+sends a request to the external projects to merge their PRs.
 
-An overlay is a file which defines where to look for the patched
-version so that testing is possible.
-The name of your overlay file should start with a five-digit pull request
-number, followed by a dash, anything (for instance your GitHub nickname
-and the branch name), then a `.sh` extension (`[0-9]{5}-[a-zA-Z0-9-_]+.sh`).
+An overlay file specifies the external PRs that should be applied during CI.
+A single overlay can cover multiple external projects.  Create your
+overlay file in the `dev/ci/user-overlays` directory.
+The name of the overlay file should start with a five-digit pull request
+number, followed by a dash, anything (by convention, your GitHub nickname
+and the branch name), then an `.sh` extension (`[0-9]{5}-[a-zA-Z0-9-_]+.sh`).
 
-This file must contain one or more invocation of the `overlay` function:
+The file must contain a call to the `overlay` function for each
+affected external project:
 ```
 overlay <project> <giturl> <ref> <prnumber> [<prbranch>]
 ```
 Each call creates an overlay for `project` using a given `giturl` and
 `ref` which is active for `prnumber` or `prbranch` (`prbranch` defaults
 to `ref`).
 
-Example of an overlay for the project `elpi` that uses the branch `noinstance`
-from the fork of `SkySkimmer` and is active for pull request `13128`
+For example, an overlay for the project `elpi` that uses the branch `noinstance`
+from the fork of `SkySkimmer` and is active for pull request `13128`:
 ```
 overlay elpi https://github.com/SkySkimmer/coq-elpi noinstance 13128
 ```
 
-Such a file can be created automatically using the scripts
+The github URL and base branch name for each external project are listed in
+[`ci-basic-overlay.sh`](../ci-basic-overlay.sh).  For example, the entry for
+`elpi` is
+```
+project elpi "https://github.com/LPCIC/coq-elpi" "coq-master"
+```
+But substitute the name of your fork into the URL, e.g. `SkySkimmer/coq-elpi`
+rather than `LPCIC/coq-elpi`.  Use `#` to mark any comments.
+
+Overlay files can be created automatically using the script
 [`create_overlays.sh`](../../dev/tools/create_overlays.sh).
-See also the list of projects for which one can write an overlay in
-the file [`ci-basic-overlay.sh`](../ci-basic-overlay.sh).
 
 ### Branching conventions
 
 We suggest you use the convention of identical branch names for the
 Coq branch and the CI project branch used in the overlay. For example,
-if your Coq PR is coming from the branch `more_efficient_tc`, and that
-breaks `ltac2`, we suggest you create a `ltac2` overlay with a branch
+if your Coq PR is in your branch `more_efficient_tc` and
+breaks `ltac2`, we suggest you create an `ltac2` overlay with a branch
 named `more_efficient_tc`.
+
+### Typical workflow
+
+- Observe that your changes breaks some external projects in CI.
+- For each broken project, run `make <job name>`, e.g. `make ci-elpi`,
```

Comment:
Don't forget to build your change locally first ;) and note that if building the stdlib with dune you should ensure that _build_vo does not exist (or after #14364 just call `make -f Makefile.dune ci-foo`)

Reply:
"if building the stdlib with dune" - you mean "if you build Coq with dune", right?

Reply:
the stdlib of coq yes